src/attestation/azext_attestation/tests/latest/test_attestation_scenario.py

```diff
@@ -0,0 +1,116 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+import os
+from .. import try_manual, raise_if
+from azure.cli.testsdk import JMESPathCheck
+from azure.cli.testsdk import JMESPathCheckExists
+from azure.cli.testsdk import NoneCheck
+from azure.cli.testsdk import ResourceGroupPreparer
+from azure.cli.testsdk import ScenarioTest
+
+
+TEST_DIR = os.path.abspath(os.path.join(os.path.abspath(__file__), '..'))
+
+
+@try_manual
+def setup(test, rg):
+    pass
+
+
+# EXAMPLE: AttestationProviders_Create
+@try_manual
+def step_attestationproviders_create(test, rg):
+    test.cmd('az attestation create '
+             '--name "{myattestation}" '
+             '--resource-group "{rg}" '
+             '--location "eastus2"',
+             checks=[
+                 JMESPathCheck('name', test.kwargs.get('myattestation', '')),
+                 JMESPathCheck('resourceGroup', rg),
+                 JMESPathCheck('location', 'eastus2')])
+
+
+# EXAMPLE: AttestationProviders_Get
+@try_manual
+def step_attestationproviders_get(test, rg):
+    test.cmd('az attestation show '
+             '--name "{myattestation}" '
+             '--resource-group "{rg}"',
+             checks=[
+                 JMESPathCheck('name', test.kwargs.get('myattestation', '')),
+                 JMESPathCheck('resourceGroup', rg),
+                 JMESPathCheck('location', 'eastus2')
+             ])
+
+
+# EXAMPLE: AttestationProviders_List
+@try_manual
+def step_attestationproviders_list(test, rg):
+    test.cmd('az attestation list '
+             '--resource-group=',
+             checks=[
+                 JMESPathCheckExists('value[?name==\'{}\']'.format(
+                     test.kwargs.get('myattestation', '')))
+             ])
+
+
+# EXAMPLE: AttestationProviders_ListByResourceGroup
+@try_manual
+def step_attestationproviders_listbyresourcegroup(test, rg):
+    test.cmd('az attestation list '
+             '--resource-group "{rg}"',
+             checks=[
+                 JMESPathCheck('value[0].name',
+                               test.kwargs.get('myattestation', ''))
+             ])
+
+
+# EXAMPLE: AttestationProviders_Delete
+@try_manual
+def step_attestationproviders_delete(test, rg):
+    test.cmd('az attestation delete '
+             '--name "{myattestation}" '
+             '--resource-group "{rg}" '
+             '--yes',
+             checks=[])
+    test.cmd('az attestation list '
+             '--resource-group "{rg}"',
+             checks=[test.check('length(value)', 0)])
+
+
+@try_manual
+def cleanup(test, rg):
```

Comment:
This does nothing. Is this here because some of the decorates need to call it later?

Reply:
No need. We recommend write test like the old way like CLI.

You can comment it or delete it.

Reply:
> No need. We recommend write test like the old way like CLI.
> 
> You could comment it or delete it.

What's the old way you mean? Can you give an example?
If no need to have this function, I will delete it.

Reply:
Like what you did in https://github.com/Azure/azure-cli-extensions/blob/master/src/portal/azext_portal/tests/latest/test_portal_scenario.py

Reply:
@RodgeFu Do you know if there has any specific reason that changing the tests from the old way?

Reply:
The setup/cleanup method are there to support easier test customization. By default, they does nothing, but if needed, you can easy override them to provide extra setup/cleanup logic for test.